Smoothing code.cpp

```diff
@@ -31,6 +31,31 @@ struct MovingAverage {
 std::uint16_t get_reading() {
     // this always returns 7 just because.... put your "read the pin" code here
     return 7;
+  int sensorPin = A0;
+int ledPin = LED_BUILTIN;
+int sensorValue = 0;
+void setup() {
```

Comment:
`setup()` is a function and needs to be in the global scope as well.

put it outside the curly braces of `get_reading()`